db/migrate/20250127092542_add_flag_for_default_pairwise_de_ui.rb

```diff
@@ -0,0 +1,11 @@
+class AddFlagForDefaultPairwiseDeUi < Mongoid::Migration
+  def self.up
+    FeatureFlag.create!(name: 'default_pairwise_de_ui',
+                        default_value: false,
+                        description: 'show pairwise differential expression UI by default')
+  end
+
+  def self.down
+    FeatureFlag.find_by(name: 'default_pairwise_de_ui')&.destroy
```

Comment:
I know this is our SOP in migrations (as is evidenced by pretty much all other FF-related ones), but we ideally should be calling `FeatureFlag.retire_feature_flag('default_pairwise_de_ui')` here instead.  That also cleans up any existing flags that have been set for users/studies.  Really not a big deal, just a note for future "us".

Reply:
Nice refinement, done in 6ca4b6f.